deckhouse-controller/pkg/controller/module-controllers/release/controller.go

```diff
@@ -1138,8 +1141,6 @@ func restoreModuleSymlink(externalModulesDir, symlinkPath, moduleRelativePath st
 }
 
 type moduleValidator interface {
```

Comment:
It seems it's reasonable to rename this interface to something else ( like moduleManager ) as it has nothing to do with validations any longer.

Reply:
Fixed